rpcs3/rpcs3qt/pad_led_settings_dialog.cpp

```diff
@@ -0,0 +1,107 @@
+﻿#include "pad_led_settings_dialog.h"
+#include <QColorDialog>
+
+pad_led_settings_dialog::pad_led_settings_dialog(const int& colorR, const int& colorG, const int& colorB, const bool& led_low_battery_blink, const bool& led_battery_indicator, const int& led_battery_indicator_brightness, QDialog * parent)
+    : QDialog(parent)
+    , ui(new Ui::pad_led_settings_dialog)
+    , m_initial{colorR, colorG, colorB, led_low_battery_blink, led_battery_indicator, led_battery_indicator_brightness}
+{
+	ui->setupUi(this);
+	setFixedSize(size());
+	m_new = m_initial;
+
+	// Fill the form
+	ui->hs_indicator_brightness->setValue(m_new.battery_indicator_brightness);
+	ui->cb_led_blink->setChecked(m_new.low_battery_blink);
+	ui->cb_led_indicate->setChecked(m_new.battery_indicator);
+	switch_groupboxes(m_new.battery_indicator);
+	update_slider_label(m_new.battery_indicator_brightness);
+	connect(ui->bb_dialog_buttons, &QDialogButtonBox::clicked, [this](QAbstractButton* button) {
```

Comment:
```suggestion
	connect(ui->bb_dialog_buttons, &QDialogButtonBox::clicked, [this](QAbstractButton* button)
	{
```

Reply:
Regarding the paintEvent:

https://doc.qt.io/qt-5/qpainter.html

> Warning: When the paintdevice is a widget, QPainter can only be used inside a paintEvent() function or in a function called by paintEvent().